crates/call-parameters/src/lib.rs

```diff
@@ -14,13 +14,14 @@
  * limitations under the License.
  */
 
+#[cfg(target_arch = "wasm32")]
 use marine_macro::marine;
 
 use serde::Serialize;
 use serde::Deserialize;
 
 /// Describes an origin that set corresponding value.
-#[marine]
+#[cfg_attr(target_arch = "wasm32", marine)]
```

Comment:
I still suggest to use a `marine` feature (non-default).

Reply:
I dont want to use a feature because they cannot be disabled. If you run, for example:
`cargo build -p disabled-feature-crate -p enabled-feature-crate` the feature will be enabled for both. If there no reason to have `#[marine]` invoked on non-wasm32 target, there is no reason to avoid targeting arch. It still will be enabled for wasm-bindgen targets.

Reply:
I would also want to decouple the content of `marine-rs-sdk-main` that does not need API exports into a separate crate, so `marine-runtime` can use it without depending on `marine-rs-sdk-main`. Or put wasm-only parts of `marine-rs-sdk-main` under `cfg(target_arch = "wasm32")`

Reply:
Well, we can discuss it again on tomorrow's daly